src/ops.rs

```diff
@@ -314,6 +314,37 @@ pub trait OpTrait {
     }
 }
 
+/// Properties of child graphs of ops, if the op has children.
+#[enum_dispatch]
+pub trait OpParent {
+    /// The inner function type of the operation, if it has a child dataflow
+    /// sibling graph.
+    fn inner_function_type(&self) -> Option<FunctionType> {
+        None
+    }
+}
+
+impl<T: DataflowParent> OpParent for T {
+    fn inner_function_type(&self) -> Option<FunctionType> {
+        Some(DataflowParent::inner_signature(self))
+    }
+}
+
+impl OpParent for Module {}
```

Comment:
As I understand it, `enum_dispatch` requires that we implement OpParent (somehow) for basically anything that we use as an Op, so although these are a minor annoyance, there's little opportunity for error here. (Right?)

Reply:
right, missing one will cause an error